### DISCUSSION TITLE
Only 9999 results are returned by the user search API

### DISCUSSION THREAD

**Discussion by komalm** [Sunbird-Lern/Community#88] [Issues] [answered]
Hello @reshmi-nair, @Hari-stackroute
Hope you are doing well!..

**Site Environments Details** - SunbirdEd portal of version 4.4.1
I am referring the documentation of user search API [here](https://lern.sunbird.org/learn/product-and-developer-guide/user-and-org-service/api-documentation/user-management)

**Issue -** 

Only 9999 results are returned by the user search API. Yet, we needed more (actually All)

Consider the following scenario: 
1. Total number of users registered with the XYZ organization
2. Count of users is selected board PQR
etc. So how do we go about getting that?

cc @Renuka-s, @vinod303

**Comment by reshmi-nair** [comment, marked as answer]
@komalm Search API will not be able to return more rows until we change the default config of Elastic Search. And returning such huge data is not recommended from API side. For your use cases, its better to write scala scripts.
